meinberlin/config/settings/base.py

```diff
@@ -267,8 +268,6 @@
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_EMAIL_VERIFICATION = "mandatory"
 ACCOUNT_FORMS = {"signup": "meinberlin.apps.users.forms.TermsSignupForm"}
-ACCOUNT_LOGIN_ATTEMPTS_LIMIT = 10
```

Comment:
This is deprecated and now is set via `ACCOUNT_RATE_LIMITS`, the default for failed logins is "10/m/ip,5/5m/key", so 10 failed logins per minute per ip and 5 failed logins per 5 minutes per account vs 10 logins per 5 mins with the old setting. Do you think we need to change the default or are we okay?

Reply:
It's good.